### DISCUSSION TITLE
Include migration in the docker-compose.yml

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#22715] [Feature Requests]
### Summary

I added the following command in my docker-compose.yml
```
  migrate:
    image: directus/directus:latest
    command: npx directus database migrate:latest
```

The I run 
`$ docker compose run migrate`

But get this error 
`"select * from `directus_migrations` order by `version` asc - SQLITE_ERROR: no such table: directus_migrations"`

which reveals that the migration latest is not run properly from 10.10.5 to 10.12.1

It would be nice to have in the example
image: directus/directus:latest 
and be able to integrate the migration.

Or to tell how to run npx directus database migrate:latest in a container that is always restarting because of the following:
`ERROR: select * from `directus_migrations` order by `version` asc - SQLITE_ERROR: no such table: directus_migrations`

Don't know if it is a bug, so I post here as a discussion.

### Motivation

Go easily from one version to the latest

### Detailed Design

Implementing in docker-compose.yml

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

none

### Alternatives

export locally, run the command locally, the import. But that's a lot.

### Adoption Strategy

Easy

**Comment by br41nslug** [comment]
Running migrations is already included in the docker image all you have to do for an update is update your image to the latest version from `directus/directus:10.X.X` to `directus/directus:10.12.X` and restart (don't rely on the `latest` tag with docker https://vsupalov.com/docker-latest-tag/)

Running migrations wont get you out of a previously created database error state tho.
